apps/web/src/services/auth/type.ts

```diff
@@ -1,13 +1,23 @@
+import type { ComputedRef } from 'vue';
 import type { TranslateResult } from 'vue-i18n';
 
+export interface PasswordFormState {
+    userIdInput: ComputedRef<string>,
+    passwordInput: ComputedRef<string>,
+    confirmPasswordInput: ComputedRef<string>,
+}
+
 export interface PasswordFormValidation {
-    isIdValid: undefined | boolean,
-    idInvalidText: TranslateResult | string,
-    isPasswordValid: undefined | boolean,
-    isConfirmPasswordValid: undefined | boolean,
-    confirmPasswordInvalidText: TranslateResult | string,
+    isIdValid?: boolean,
+    idInvalidText: TranslateResult,
 }
 
 export interface PasswordFormExpose {
     validationState: PasswordFormValidation
 }
+
+export const PasswordStatus = {
```

Comment:
nitpick: Please use `SNAKE_CASE` to object constant :)

```suggestion
export const PASSWORD_STAUTS = {
```